tests/lib/processor.js

```diff
@@ -115,7 +115,7 @@ describe("processor", () => {
                 "```js",
                 "backticks",
                 "```",
-                "~~~javascript",
+                "~~~js",
```

Comment:
Maybe it would make sense to leave this as "javascript" to ensure it gets translated into `.js`?

Reply:
There are already the preexisting test cases "should find code fences with javascript info string" and "should find code fences with node info string" which test the mapping. As such I changed it in the other test cases.